src/public/packages/backpack/crud/css/list.css

```diff
@@ -54,7 +54,8 @@
 	}
 
   /*  DataTables Loading State Visual Improvements  */
-
+	table.dataTable.dtr-inline.collapsed>tbody>tr[role="row"]>td.dtr-control:before, 
+	table.dataTable.dtr-inline.collapsed>tbody>tr[role="row"]>th.dtr-control:before,
```

Comment:
This is to future-proof it, it's not _actually_ needed, it will be needed in 4.2 when we get to actually hide the BulkActions column on XS.

So it's not a breaking change, even if it's a change to `list.css`